### DISCUSSION TITLE
Grouped ListViews not working on Skia?

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#12698] [Q&A] [answered]
Before I go too far down the rabbit hole, should grouped listviews on Skia work? I have pulled my hair trying to just have simple group headers, but nothing gets rendered, it's just a plain list of the items (not grouped, no group header). Currently I can't run my WPF or Windows heads (unrelated), so I can't verify if it works there, but I wanted to see if it's supposed to work (or if there are examples)?

**Comment by HakanL** [comment]
I've added a simple example here: https://github.com/HakanL/Uno-Navigate-Issue

**Reply by jeromelaban** [reply, marked as answer]
Indeed, grouping does not yet work on Skia and Wasm, we're tracking this in https://github.com/unoplatform/uno/issues/234

**Reply by HakanL** [reply]
I see. Considering this was opened in 2018, I guess I'm not going to hold my breath on this one? :)

**Reply by jeromelaban** [reply]
We prioritize issues both on community demand and paid client requests; process described here [Sustaining the Open-Source Uno Platform](https://platform.uno/blog/sustaining-the-open-source-uno-platform/). Please reach out to us at info@platform.uno if you'd like to learn more.

**Reply by HakanL** [reply]
Makes sense, wasn't expecting you to change any priorities, was just curious if it's on the roadmap. I can solve my issue by just different templates, but wanted to know the current status.

**Reply by francoistanguay** [reply]
Have you explored using ItemsRepeater? 
Something like this: https://github.com/unoplatform/uno/blob/master/src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Repeater/ItemsRepeater_Nested.xaml

Official docs: https://learn.microsoft.com/en-us/windows/apps/design/controls/items-repeater#display-grouped-items
